admin_manual/installation/system_requirements.rst

```diff
@@ -87,8 +87,8 @@ Desktop
 - CentOS 6 & 7 (64-bit only)
 - Debian 8.0 & 9.0
 - Fedora 27 & 28
-- Ubuntu 16.04 & 18.04
-- openSUSE Leap 42.3
+- Ubuntu 16.04 & 17.10 & 18.04
+- openSUSE Leap 42.3 & 15.0
```

Comment:
Looks fine, based on https://en.opensuse.org/openSUSE:Roadmap. However, Ubuntu 17.10 is, as @phil-davis said, EOL.

Reply:
My EOL date collection is in https://github.com/owncloud/ownbrander/blob/master/brand-items.php#L1661
Note, that for desktop clients, the policy is to support the last two releases. That includes Ubuntu 18.04 and 17.10 -- technically that is possible as all the dependencies are cached in OBS, even after the upstream 17.10 goes away.
(Ubuntu 17.04 should not be in the list anymore, but there was no deprecation warning with last client release, so we did the deprecation warning with client 2.4.2 and will execute the deprecation with client 2.5.0)